src/simplewallet/simplewallet.cpp

```diff
@@ -1832,7 +1832,7 @@ bool simple_wallet::show_qr_code(const std::vector<std::string> &args)
   WTEXTON();
   try
   {
-    const std::string address = "monero:" + m_wallet->get_subaddress_as_str({m_current_subaddress_account, subaddress_index});
+    const std::string address = "aeon:" + m_wallet->get_subaddress_as_str({m_current_subaddress_account, subaddress_index});
```

Comment:
Just a matter of taste, but it could have been done in the first cherry-pick commit. But OK.